source/how-tos/boot-acrn.rst

```diff
@@ -0,0 +1,357 @@
+.. _boot-acrn:
+
+Boot ACRN Hypervisor
+------------------------
+
+Step-by-Step guide to build a custom |SPN| to boot ACRN and provision eMMC on |UP2|.
+
+Prerequisites
+^^^^^^^^^^^^^^^^^
+
+Before you start, prepare the following items:
+
+* ACRN images for |UP2| - URL TBD
+
+* Serial debug adaptor for UART0 on |UP2| - Required to trigger fastboot and perform firmware update
+
+* Micro USB cable (type A to microB)
+
+* USB key
+
+* **RECOMMENDED:** A SPI programmer in case you brick |UP2| and need to restore factory BIOS. See |UP2_FLASHING| for details.
+
+.. |UP2_FLASHING| raw:: html
+
+   <a href="https://wiki.up-community.org/BIOS_chip_flashing_on_UP_Squared" target="_blank">BIOS chip flashing</a>
+
+
+For more details on setting up the hardware, see :ref:`up2-board`.
+
+
+Build |SPN| 
+^^^^^^^^^^^^^^^^
+
+Build and stitch a custom |SPN| with fastboot support
+
+**Step 1:** Install tool chains and clone SBL source code from GitHub
+
+  Run::
+
+    sudo apt-get install -y build-essential iasl python uuid-dev nasm openssl gcc-multilib qemu
+    git clone git@github.com:slimbootloader/slimbootloader.git
+    cd slimbootloader
+    git config --global user.email "email@example.com"
+    git config --global user.name "your name"
+
+  Compile |SPN| just to verify your setup is complete::
+
+    python BuildLoader.py build apl
+
+
+**Step 2:** Download additional required images and tools 
+
+1. Download fastboot executable ``fastboot.elf`` from |FB| to |SPN| source directory
+
+.. |FB| raw:: html
+
+   <a href="https://github.com/intel/kernelflinger/raw/master/prebuilt/board/APL_UP2/fastboot.elf" target="_blank">here</a>
+
+2. Download |UP2| BIOS from |BIOS_V4| and unzip as is in |SPN| source directory
+
+.. |BIOS_V4| raw:: html
+
+   <a href="https://downloads.up-community.org/download/up-squared-uefi-bios-v4-0/" target="_blank">here</a>
+
+3. Download IAS script ``iasimage`` from |IAS| to |SPN| source directory
+
+.. |IAS| raw:: html
+
+   <a href="https://raw.githubusercontent.com/intel/iasimage/master/iasimage" target="_blank">here</a>

```

Comment:
We should probably direct the user to install iasimage using the instructions provided by *that* repository, in case the instructions ever change.

Reply:
IAS how-to now merged